webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -4100,6 +4101,116 @@ const firstVisibleMsg = {
                 }
                 $mck_msg_to.focus();
             };
+
+            _this.isWithinBusinessHours = function (team) {
+                // don't show message if business hour setting are not set
+                if (!team || !team.businessHourMap || !team.timezone) {
+                    return true;
+                }
+                const convertToMinutes = (timeStr) => {
```

Comment:
there must be some existing code for the same

Reply:
Elsewhere, we're displaying the time directly using the local Date object. However, this function is specifically used to handle the time format in which the backend stores the business hour map.